### PR TITLE
S3t1 item counter component

### PR DESCRIPTION
**Pull request by Steph-ski**
create the JSX

when you increment/decrement it manages order state in the OrderPage Component

### CODE REVIEW

File: src/ItemCounter/index.js [hunk 1]
```diff
@@ -1,12 +1,38 @@
...
+    }
+
+    const handleClickMinus = () => {
+        let tempOrder = {...order}
+
+        if (index in tempOrder) {
+            if (tempOrder[index] === 1) {
+                delete tempOrder[index]
+            } else {
+                tempOrder[index] = parseInt(tempOrder[index]) - 1
```
Comment (alreuben) [inline src/ItemCounter/index.js:21]: line 7 has no whitespace after [index] but has some after the '='. Also uses whitespace here - should probably decide if we want to remove or add whitespace to all.

Also whitespace after the '-' but not after the '+' on line 7

File: src/ItemCounter/index.js [hunk 2]
```diff
@@ -1,12 +1,38 @@
+const ItemCounter = ({order, setOrder, index}) => {
+
+    const handleClickPlus = () => {
+        let tempOrder = {...order}
+
+        if (index in tempOrder){
```
Comment (alreuben) [inline src/ItemCounter/index.js]: add space before {

File: src/MenuItem/index.js [hunk 3]
```diff
@@ -1,7 +1,7 @@
 import './styles.scss';
 import ItemCounter from "../ItemCounter";
 
-const MenuItem = ({menuItemData}) => {
+const MenuItem = ({menuItemData,order,setOrder,index}) => {
```
Comment (alreuben) [inline src/MenuItem/index.js]: add spaces after commas

**Review by alreuben — CHANGES_REQUESTED** [review]

**Review by Yas-Shen — COMMENTED** [review]
I think Ayannah pretty much summed it up, can't see anything else on my end.

**Review by alreuben — APPROVED** [review]

**Pull request merged**